blog/2015-02-27-Docker-And-Wildfly-2.adoc

```diff
@@ -0,0 +1,115 @@
+= Docker and Wildfly - Part Two
+:page-layout: blog
+:page-author: rstryker
+:page-tags: [jbosstools, devstudio, server, docker, wildfly]
+
+If you've already link:2015-02-26-Getting-Started-With_Docker-And-Wildfly.html[gotten started] with trying out 
+Docker and JBoss Tools, and want to see what other options are available, this blog post will  
+explaini how to run Wildfly in a Docker container and configure it for management tasks and
+managed deployments. 
+
+=== Customizing your Dockerfile
+
+Since the default `jboss/wildfly` Docker configuration isn't good enough for us, we'll need
+to customize it, and this means writing your very first Dockerfile. Our goals here are 
+to add a management user to the Dockerfile, and also to expose the management ports when
+running the container. 
+
+First, you'll want to make a folder on your filesystem that you can play around in. 
+I'll name mine `docker_jbds_wf_mgmt` - 
+Inside this folder, we'll make a new file named `Dockerfile` and give it the following content:
+
+....
+FROM jboss/wildfly:latest
+
+USER jboss
+RUN /opt/jboss/wildfly/bin/add-user.sh admin Admin#70365 --silent
+CMD ["/opt/jboss/wildfly/bin/standalone.sh", "-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
+....
+
+You can see that we're extending the original `jboss/wildfly` container. 
+In the above `RUN` command, we're also adding a management user. Feel free
+to customize the password as you wish.  
```

Comment:
hmm - thats not nice having to expose that in dockerfile...

Maybe note users should be aware this should not be done in public dockerfiles ;)